REQUEST_EXAMPLES.md

```diff
@@ -83,13 +66,78 @@ Adding ',', ':' or '-' anywhere and the casing of the sentence in the tweet text
  ```
  "Please provide 'Ambient temperature' in the format - 273.15K."
  ```
+
+## Commom optional keywords -
  - Providing `experiment`.
- Required only when you want to simulate an experiment. For example -
+ Required only when you want to simulate an experiment. Casing of the exepriment should be correct and always use "'" to specify the conditions. For example -
  ```
- with<space or no character>experiment<space or no character>- [("Discharge at C/10 for 10 hours or until 3.3 V","Rest for 1 hour","Charge at 1 A until 4.1 V","Hold at 4.1 V until 50 mA","Rest for 1 hour")]<space>*<space>3
- use the<space or no character>experiment<space or no character>[("Discharge at C/10 for 10 hours or until 3.3 V","Rest for 1 hour","Charge at 1 A until 4.1 V","Hold at 4.1 V until 50 mA","Rest for 1 hour")]<space>*<space>3
+ with<space or no character>experiment<space or no character>- [('Discharge at C/10 for 10 hours or until 3.3 V','Rest for 1 hour','Charge at 1 A until 4.1 V','Hold at 4.1 V until 50 mA','Rest for 1 hour')]<space>*<space>3
+ use the<space or no character>experiment<space or no character>[('Discharge at C/10 for 10 hours or until 3.3 V','Rest for 1 hour','Charge at 1 A until 4.1 V','Hold at 4.1 V until 50 mA','Rest for 1 hour')]<space>*<space>3
  ```
  If not provided or if incorrectly provided, the bot will give the following error -
  ```
  "Please provide 'Ambient temperature' in the format - 273.15K."
  ```
+
+## Compare 2 or more models with a constant discharge or with an experiment
+### Some examples -
+```
+@battbot_ compare SPM and SPMe for a constant dicharge of 1.25C at 290K with Chen2020 parameters #battbot.
+@battbot_ #battbot can you compare Single particle model, Single particle model with electrolyte and DFN model at 300K with a c-rate of 0.5C with Marquis2019 chemistry?
+@battbot_ #battbot compare spm, spme DFN model with a constant dicharge of 0.75C with Ai2020 at 280K
+@battbot #battbot compare spm and spme with chen2020 parameters with the experiment - [("Discharge at C/10 for 10 hours or until 3.3 V","Rest for 1 hour","Charge at 1 A until 4.1 V","Hold at 4.1 V until 50 mA","Rest for 1 hour")] * 3 at 298K
+```
+ - Adding the keyword `"Compare"`.
+ For example -
+ ```
+ <space or no character>Compare<space or no character>
+ Compare SPM and SPMe
+ ```
+ If not provided, the bot will give the following error -
+ ```
+ "I'm sorry, I couldn't understand the requested simulation."
+ ```
+
+## Compare 2 or more values for a parameter with a constant discharge or with an experiment
+
+Always provide the keywords in the following order -
```

Comment:
Could you make it clear that the keywords are "vary" and "values", and then you can provide either constant discharge or experiment. At the moment it could be interpreted that you need to include the text "Parameter to vary" and "varied values"